src/components/organisms/Table/Sortable/helpers.tsx

```diff
@@ -0,0 +1,36 @@
+import { SortableTableRow, SortableTableSortArgs } from './SortableTable'
+
+// H E L P E R S
+
+export const getCellValue = (row: SortableTableRow, field: string): boolean | number | string => {
+  const value = row[field]
+  if (Array.isArray(value)) return ''
+  if (Number.isInteger(value)) return value
+  if (typeof value === 'string') return value.toLowerCase()
+  return ''
+}
+
+export const handleSort = (sortArgs: SortableTableSortArgs): void => {
+  const { field, sortDirection, setSortData, originalData, sortData } = sortArgs
+
+  if (sortDirection === null) {
+    setSortData(originalData)
+    return
+  }
+
+  const newData = [...sortData]
+  let sortedData: SortableTableRow[] = []
+
+  sortedData = newData.sort((firstItem, secondItem) => {
+    const firstValue: string | boolean | number = getCellValue(firstItem, field)
```

Comment:
The return type of `getCellValue` is already  `boolean | number | string` so you don't need to respecify them but good point to think about typescript types 👍